pennylane/pulse/transmon.py

```diff
@@ -57,7 +57,7 @@ def transmon_interaction(
         + \sum_{(i, j) \in \mathcal{C}} g_{ij} \left(b^\dagger_i b_j + b_j^\dagger b_i \right)
         + \sum_{q\in \text{wires}} \alpha_q b^\dagger_q b^\dagger_q b_q b_q
 
-    where :math:`[b^\dagger_p, b_q] = i \delta_{pq}` are creation and annihilation operators.
+    where :math:`[b_p, b_q^\dager] = \delta_{pq}` are creation and annihilation operators.
```

Comment:
Was this a sign error combined with a single typo (including the `i`)?
```suggestion
    where :math:`[b_p, b_q^\dagger] = \delta_{pq}` are creation and annihilation operators.
```

Reply:
I dont know where this came from, probably just confused and no one noticed 😅